src/containers/SideBar/components/VersionChip/VersionChip.tsx

```diff
@@ -0,0 +1,24 @@
+import { useTranslation } from 'react-i18next';
+import ConnectedPulse from './ConnectedPulse/ConnectedPulse';
+import { Divider, Wrapper } from './styles';
+import { useAppStateStore } from '@app/store/appStateStore';
+
+interface Props {
+    version: string;
+}
+
+export default function VersionChip({ version }: Props) {
+    const { t } = useTranslation('common', { useSuspense: false });
+
+    const isAppSettingUp = useAppStateStore((s) => s.isSettingUp);
```

Comment:
re your question - yeah this should be `base_node.is_connected` instead, already stored in `useMiningStore` :D example: used here in [`DebugSettings.tsx`](https://github.com/tari-project/universe/blob/68fa3c6711f9e0613ba5c12397d85fbf8a6c10f6/src/containers/Settings/sections/experimental/DebugSettings.tsx#L18)